source/install/install-kubernetes-mattermost.rst

```diff
@@ -61,7 +78,98 @@ Save the file as ``mattermost-installation.yaml``:
       username: ""
       password: ""
 
-**3. Edit the Installation Manifest File**
+Save the file as ``mattermost-installation.yaml``.
+
+**3. Apply the Installation Manifest File**
+
+To initiate deployment, apply the file, specifying the correct path, using:
+
+.. code-block:: sh
+
+  $ kubectl create ns mattermost
+  $ kubectl apply -n mattermost -f /path/to/mattermost-installation.yaml
+
+The deployment process can be monitored in the Kubernetes user interface.
+
+**4. Configure DNS and Use Mattermost**
+
+When the deployment is complete, obtain the hostname or IP address of your Mattermost deployment using the following command:
+
+.. code-block:: sh
+
+  $ kubectl -n mattermost get ingress
+
+Copy the resulting hostname or IP address from the ``ADDRESS`` column, open your browser, and connect to Mattermost.
+
+Use your domain registration service to create a canonical name or IP address record for the ``ingressName`` in your manifest,
+pointing to the address you just copied. For example, on AWS you would do this within a hosted zone in Route53.
+
+Navigate to the ``ingressName`` URL in your browser and use Mattermost.
+
+Restoring an Existing Mattermost MySQL Database
+-----------------------------------------------
+
+You can leverage the Mattermost Operator to utilize an existing Mattermost MySQL database with a new Mattermost installation, in its own namespace.
+The steps you follow to create and upload your backup depends on the provider you're using and your use case. It's
+recommended that you consult the relevant documentation or, if your deployment is managed in a different way, consult your Administrator.
+
+It is important to note that this process requires the creation of a new Mattermost
```

Comment:
This needs to be much more explicit about this being a "new" installation as such. Is there a situation where this can be applied to an existing cluster installation file at all? Or would the customer always have to create a new namespace/install yaml? Not sure how much detail we want to go into about a use case where a customer could spin this up in a new namespace and then take down the other instance (if that would be relevant/accurate).

Reply:
The gist of it is that the restore is meant for the creation time of the installation, not for an existing installation. As long as that's clear I think we're good.

Reply:
That makes sense but I don't think the docs are clear on that. 

So, to clarify, would this instance be the de facto one going forward and any current instances/deployments would be taken down once this is up and running? Or could they run in tandem - not sure why that would happen unless it's to ensure all is running correctly but in that case wouldn't blue-green work better?

Reply:
@jwilander, I believe backup and restore was also for possible rollback of the DB in case of a large issue.

For instance, should the docs also highlight that this could be used on an existing mattermost installation to rollback the DB without creating a new installation entirely?

Reply:
If that's the use case, which makes sense, would it be better if we moved this to the section with Canary Builds and Blue-green as another management option vs. a discrete process? Then it may have more context.

Reply:
I believe the location of this information is fine, but I would suggest that we change the wording a bit. Generally, I see the backup and restore process having two very common use-cases:
1. What is outlined here. You had Mattermost running and want to start using it in k8s with our operator. To do so, you create a new installation and restore the data like is described in the PR already.
2. You have backups going for your operator-run mattermost server. One day something happens and you want to revert the database to X days ago, so you use this backup and restore process to do just that on the existing installation.

That seems correct, right @jwilander? I could have sworn we wanted both options.

@justinegeffen, I am also completely fine with that coming in a follow-up PR. The info presented here isn't wrong by any account.

Reply:
That's very helpful feedback, thank you! I've edited the introduction to include both aspects.